doc/arkode/guide/source/Usage/User_callable.rst

```diff
@@ -4685,3 +4685,26 @@ rescale the upcoming time step by the specified factor.  If a value
       * ``examples/arkode/C_serial/ark_heat1D_adapt.c``
 
    .. versionadded:: 6.1.0
+
+
+
+.. _ARKODE.Usage.SUNStepperInterface:
+
+Using an ARKODE solver as a SUNStepper
+--------------------------------------
+
+The utility function :c:func:`ARKodeCreateSUNStepper` wraps an ARKODE memory
+block as a :c:type:`SUNStepper`.
+
+.. c:function:: int ARKodeCreateSUNStepper(void *inner_arkode_mem, SUNStepper *stepper)
+
+   Wraps an ARKODE integrator as a :c:type:`SUNStepper`.
+
+   :param arkode_mem: pointer to the ARKODE memory block.
+   :param stepper: the :c:type:`SUNStepper` object.
+
+   :retval ARK_SUCCESS: the function exited successfully.
+   :retval ARK_MEM_FAIL: a memory allocation failed.
+   :retval ARK_SUNSTEPPER_ERR: the :c:type:`SUNStepper` initialization failed.
+
+   .. versionadded:: x.y.z
```

Comment:
Are there any assumed requirements about the ARKODE solver module?  Specifically, since some steppers "support" all ARKODE functionality, whereas other steppers only support a tiny subset, then it would be good to clarify which steppers this function would work for.

Reply:
The only functionality not supported by all of ARKODE is `SUNStepper_SetForcing`. I'll make note of that here and in ARKStep, ERKStep, etc.